### PR TITLE
[FIX] base/ir: Run cron manually calls _callback with SUPERUSER_ID

### PR DESCRIPTION
**Pull request by tivisse**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:
## 

I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

It shouldn't be called with the current user.

### CODE REVIEW

File: openerp/addons/base/ir/ir_cron.py [hunk 1]
```diff
@@ -87,7 +87,7 @@ def method_direct_trigger(self, cr, uid, ids, context=None):
             context={}
         cron_obj = self.browse(cr, uid, ids, context=context)
         for cron in cron_obj:
-            self._callback(cr, uid, cron_obj.model, cron_obj.function, cron_obj.args, cron_obj.id)
+            self._callback(cr, SUPERUSER_ID, cron_obj.model, cron_obj.function, cron_obj.args, cron_obj.id)
```
Comment (guewen) [inline openerp/addons/base/ir/ir_cron.py]: Each `ir.cron` record has an associated user with which the cron is run. Shouldn't it be used here?

Comment (lmignon) [inline openerp/addons/base/ir/ir_cron.py]: I share the same POV as @guewen. Moreover using the SUPERUSER_ID breaks the multi company rules based on record rules. For sure this code will introduce bugs!

Comment (guewen) [inline openerp/addons/base/ir/ir_cron.py]: Yep, we often need to duplicate a cron and assign them to different users assigned to different companies so record rules and properties are respected.

Comment (tivisse) [inline openerp/addons/base/ir/ir_cron.py]: Hello, 

Indeed that was a mistake (That you reported even before I tested the branch ^^)

Thank you.

**Pull request merged**